config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
```

Comment:
so now our manager pod is configured using this CR?

Reply:
yes